.github/workflows/ci.yml

```diff
@@ -51,9 +51,11 @@ jobs:
           sudo mv nf-test /usr/local/bin/
 
       - name: Run nf-test
+        continue-on-error: ${{ matrix.NXF_VER == 'latest-everything' }}
```

Comment:
The latest version of nextflow (25.03 or third preview) makes breaking changes to behaviour of publishing files (see https://www.nextflow.io/docs/latest/migrations/25-04.html#migrating-to-25-04-preview). This means tests with `latest-everything` will now always fail.

Added `continue-on-error` so that the tests will run, but still succeed if `latest-everything` fails. This is derived from how the latest GitHub actions works on the nf-core templates.